build.rs

```diff
@@ -36,41 +36,29 @@
 // SOFTWARE.
 
 extern crate gcc;
-use std::env;
-use std::process::Command;
-use std::collections::HashMap;
+use std::path::PathBuf;
 
-fn run_lsb_release(args: Vec<&str>) -> HashMap<String, String> {
-    let output = Command::new("lsb_release")
-                         .args(args)
-                         .output()
-                         .unwrap();
-    assert!(output.status.success());
-    let output_s = String::from_utf8(output.stdout).unwrap();
+const FEATURE_CHECKS_PATH: &str = "feature_checks";
 
-    let mut res = HashMap::new();
-    for line in output_s.lines() {
-        let parts: Vec<&str> = line.split(":").collect();
-        assert!(parts.len() == 2);
-        res.insert(String::from(parts[0].trim()), String::from(parts[1].trim()));
-    }
-    res
+/// Simple feature check, returning `true` if we have the feature.
+///
+/// The checks themselves are in files under `FEATURE_CHECKS_PATH`.
+fn feature_check(filename: &str) -> bool {
+    let mut path = PathBuf::new();
+    path.push(FEATURE_CHECKS_PATH);
+    path.push(filename.clone());
```

Comment:
I don't think you need the `clone` here?

Reply:
correct.